src/modules/EventHub/models/Event.model.ts

```diff
@@ -0,0 +1,52 @@
+import type { Event, CreateEventInput, UpdateEventInput } from '../types';
+
+const findById = async (_id: number): Promise<Event | null> => {
+  return null;
+};
+
+const list = async (
+  _page: number,
+  _limit: number
+): Promise<{ items: Event[]; total: number }> => {
+  return { items: [], total: 0 };
```

Comment:
Maybe add a TODO or throw an error here to make sure it’s clear that the implementation isn’t finalized. Otherwise, it's confusing